xds/src/main/java/io/grpc/xds/XdsComms.java

```diff
@@ -88,21 +218,50 @@ public void onCompleted() {
     boolean cancelled;
     boolean closed;
```

Comment:
`onCompleted()` should probably be translated into `onError()`.

Reply:
Done.